adoc/chapters/programming_interface.adoc

```diff
@@ -10187,7 +10187,7 @@ If a device supports concurrent access to shared allocations, then
 Additionally, users may use the [code]#mem_advise# member function to annotate
 shared allocations with [code]#advice#.
 Valid [code]#advice# is defined by the device and its associated backend.
-See <<table.members.queue>> and <<table.members.handler.copy>> for more
+See <<table.queue.shortcuts>> and <<table.members.handler.copy>> for more
```

Comment:
Thanks for fixing this.  However, the same error exists in the paragraph above too.  Can you fix that one also?

Reply:
Updated the link for prefetch method as well.